gsk4/src/auto/color_node.rs

```diff
@@ -0,0 +1,49 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files.git)
+// DO NOT EDIT
+
+use crate::RenderNode;
+use glib::translate::*;
+use glib::StaticType;
+use std::fmt;
+
+glib::wrapper! {
+    #[doc(alias = "GskColorNode")]
+    pub struct ColorNode(Shared<ffi::GskColorNode>);
+
+    match fn {
+        ref => |ptr| ffi::gsk_render_node_ref(ptr as *mut ffi::GskRenderNode),
+        unref => |ptr| ffi::gsk_render_node_unref(ptr as *mut ffi::GskRenderNode),
+    }
+}
+
+impl glib::StaticType for ColorNode {
+    fn static_type() -> glib::Type {
+        unsafe { from_glib(ffi::gsk_color_node_get_type()) }
+    }
+}
+
+impl ColorNode {
+    #[doc(alias = "gsk_color_node_new")]
+    pub fn new(rgba: &gdk::RGBA, bounds: &graphene::Rect) -> ColorNode {
```

Comment:
I think clippy would complain here that we pass `Copy` types by reference?

Reply:
(I think passing by reference here is the right thing to do)

Reply:
> I think clippy would complain here that we pass `Copy` types by reference?

Somehow it doesn't, no idea why :)

Reply:
Good then!